php/class-customize-snapshot-manager-back-compat.php

```diff
@@ -106,7 +101,7 @@ public function enqueue_controls_scripts() {
 			'initialServerDate' => current_time( 'mysql', false ),
 			'initialServerTimestamp' => floor( microtime( true ) * 1000 ),
 			'theme' => $this->original_stylesheet,
-			'previewingTheme' => isset( $preview_url_query_vars['theme'] ) ? $preview_url_query_vars['theme']: '',
+			'previewingTheme' => ! $this->customize_manager->is_theme_active(),
```

Comment:
@westonruter This is breaking my condition here https://github.com/xwp/wp-customize-snapshots/blob/9ca3ff2f90df27da5fc62946b7857b61dbcc3184/js/compat/customize-snapshots.js#L278
There are several use cases where the button has to be enabled or disabled.

1.  When they switch theme ( Has to enable ) ( Working )
2. After saving the previewing theme, reload the page, ( Has to disable ) ( Not working now )
3. After saving the previewing theme they switch to a third theme ( Has to enable ) ( Working )
4. After travelling through the themes and saving the last one, they come back to the original theme ( Should be enable ) ( Not working now ).

I was not able to satisfy these conditions with only boolean value of `previewingTheme `.